### DISCUSSION TITLE
export/print - default selection

### DISCUSSION THREAD

**Discussion by istoOi** [kimai/kimai#2555] [Questions & Answers]
Hi,

i'm trying to figure out how to set which checkboxes are selected by default in the print view.
Is there a global or per user setting? 
If it's a template, which file do i need to modify and copy to var/export/?

**Comment by kevinpapst** [comment]
https://github.com/kevinpapst/kimai2/blob/master/templates/export/renderer/default.html.twig

**Reply by istoOi** [reply]
thx.
if i copy it over to var/export/ as "custom_default.html.twig" (and reset cache), will there be selection in the print dialog?
